_maps/map_files/BoxStation/BoxStation.dmm

```diff
@@ -53470,14 +53470,14 @@
 	},
 /obj/item/clothing/head/russobluecamohat,
 /obj/item/clothing/head/russobluecamohat,
-/obj/item/clothing/head/helmet/rus_ushanka,
-/obj/item/clothing/head/helmet/rus_ushanka,
-/obj/item/clothing/head/helmet/rus_ushanka,
-/obj/item/clothing/head/helmet/rus_ushanka,
-/obj/item/clothing/under/mw2_russian_para,
-/obj/item/clothing/under/mw2_russian_para,
-/obj/item/clothing/under/mw2_russian_para,
-/obj/item/clothing/under/mw2_russian_para,
+/obj/item/clothing/head/ushanka,
+/obj/item/clothing/head/ushanka,
+/obj/item/clothing/head/ushanka,
+/obj/item/clothing/head/ushanka,
```

Comment:
No fun, these things have mediocre armor.

Reply:
Those have zero armor I thought?

Reply:
Was talking about the previous ones, no big deal though.